pkg/operator/ceph/csi/version.go

```diff
@@ -25,12 +25,10 @@ import (
 )
 
 var (
-	//minimum supported version is 2.0.0
-	minimum = CephCSIVersion{2, 0, 0}
+	//minimum supported version is 3.0.0
+	minimum = CephCSIVersion{3, 0, 0}
```

Comment:
I guess this answers my previous question about the 2.x driver if 3.0 is required now. But is it really required? Or can we still allow the min version to be 2.0? 

As a separate work item, to support different versions of csi we really should be running our test suites with different combinations of the csi version. I'll open an issue on that.

Reply:
with 1.4 Minimum is 3.0.0 as the ceph-csi snapshots are not backward compatible like I said earlier the snapshot created with 2.x cannot be deleted with 3.0.0. so that's why we are asking the user to delete older snapshots before the upgrade